docker-compose.yml

```diff
@@ -9,6 +9,7 @@ web:
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+  command: bundle exec unicorn -l 0.0.0.0:3000
```

Comment:
CCI is using compose, it's just that it's configured in `circle/docker-compose-template.yml` instead of the `docker-compose.yml` file we use for local compose (there's some env var substitution that needs to happen there). 
I'm seeing an issue with local compose where it tries to spin up the db from an old run: ```Recreating 22a7a45ffe_22a7a45ffe_champaign_db_1...
No such image: a7d662bede59726299cf40d589777591187007a32ec2a7c8b694abf092094c8e
``` I've had this problem before and I'll see if I can get this fixed and if that's maybe what's also happening for you.